src/tools/compiletest/src/runtest.rs

```diff
@@ -2710,6 +2712,25 @@ impl<'test> TestCx<'test> {
         fs::remove_dir(path)
     }
 
+    fn run_js_doc_test(&self) {
+        if let Some(nodejs) = &self.config.nodejs {
```

Comment:
Is `self.config` loaded from `config.toml`? I'm wondering how the path to the node executable is set.

Reply:
I took the same code I used previously. But iirc, it is from the `config.toml` file.

Reply:
Yep, and checked in bootstrap (this is a config "passed through" from bootstrap).